source/module_elecstate/module_charge/charge.h

```diff
@@ -50,7 +50,7 @@ class Charge
 
     void set_rhopw(ModulePW::PW_Basis* rhopw_in);
 
-    void init_rho(elecstate::efermi& eferm_iout, const ModuleBase::ComplexMatrix& strucFac);
+    void init_rho(elecstate::efermi& eferm_iout, const ModuleBase::ComplexMatrix& strucFac, const int& nbz, const int& bz);
```

Comment:
Please add annotations for this modified function.

Reply:
OK